packages/sdk/src/evm/common/feature-detection/resolveContractUriFromAddress.ts

```diff
@@ -36,7 +36,11 @@ export async function resolveImplementation(
   address: string,
   provider: providers.Provider,
 ): Promise<{ address: string; bytecode: string }> {
-  const bytecode = await fetchBytecode(address, provider);
+  const fetched = await Promise.all([
```

Comment:
its cleaner to destructure here instead of `fetched[0]` to access the values, you can do

`const [bytecode, beacon] = await Promise.all(....);`